### DISCUSSION TITLE
[DISREGARD] What3Words

### DISCUSSION THREAD

**Discussion by Der-Flip** [organicmaps/organicmaps#2132] [Ideas]
must've missed that earlier ,3Words is not open and T&C say nope: https://wiki.openstreetmap.org/wiki/What3words

~~When selecting a place I have a couple of options regarding the coordinates (decimal degrees, decimal seconds, alpha numerical, osm link).
Is there a chance to have What3Words as another option implemented or is that something that would need to be adressed (no pun intended) directly within the osm first?~~

**Comment by biodranik** [comment]
Isn't it a proprietary online API?

**Reply by Der-Flip** [reply]
I genuinely do not know. I'm really not familiar with their infrastructure / licensing or anything really.
I think the first thing one would need to know first is whether it's with you guys or with osm...

**Reply by Der-Flip** [reply]
actually.... I found the osm wiki on 3Words and yeah, it's not open and thus not desired within osm.... Unfortunately I can't find an option to close / delete this thread... If you have an option to do so I'd be much obliged...

**Reply by biodranik** [reply]
No worries, let others see this thread, it could be useful.

Maybe some day W3W become an open API )

May I ask you, why do you want it to be implemented? What is your use case?

**Comment by hb0nd** [comment]
[Plus Codes](https://maps.google.com/pluscodes/) are implemented and a good alternative :) when on a POI info page, click the location coordinates, and it'll switch through the available options.
